### ISSUE TITLE
springConfig doesn't work (?)

### ISSUE THREAD

**Issue by tofuness** [nkbt/react-collapse#32]
The springConfig doesn't seem to change the spring when you pass in an object.
```javascript
<Collapse
  isOpened={this.props.expanded}
  springConfig={[400, 25]}
>
  // ....
</Collapse>
```
However, passing in something like `{ stiffness: 400, damping: 25 }` works (?).

Edit: Okay I know the problem. I have the latest version of `react-motion` installed. New `spring()` now accepts an object (like the one I mentioned above) instead of `[stiffness, damping]`.

**Comment by nkbt** [comment]
Yep, react-motion@^0.4 has heaps of breaking changes. I am going to update react-collapse asap, though it might be a new major version due to possible breaking changes in configuration.

Thanks for heads up by the way!

**Comment by nkbt** [comment]
Fixed by https://github.com/nkbt/react-collapse/pull/33

**Comment by nkbt** [comment]
Published as react-collapse@2.0.0

New major release since breaking change for `springConfig` format.
No other breaking changes though. so upgrade should be easy
